LayoutTests/accessibility/text-marker/text-marker-bidi-inline-cite.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../resources/js-test.js"></script>
+<script src="../../resources/accessibility-helper.js"></script>
+</head>
+<body>
+
+    <p dir="auto" id="content">The alphabet is <cite dir="rtl">ABC DEF GHI <span dir="ltr">JKL MNO PQR STU VWX YZ</span></cite>.</p>
+
+<script>
+var output = "This test verifies that a paragraph with ltr text in English, containining a cite element with dir=\"rtl\", containing a span element with dir=\"ltr\" is read correctly by accessibility.\n\n";
+
+if (window.accessibilityController) {
+    var paragraph = accessibilityController.accessibleElementById("content").childAtIndex(0);
```

Comment:
I think this is actually the first static text element inside the paragraph, not the paragraph element.